dracut.sh

```diff
@@ -763,6 +763,7 @@ stdloglvl=$((stdloglvl + verbosity_mod_l))
 [[ $dracutbasedir ]] || dracutbasedir=$dracutsysrootdir/usr/lib/dracut
 [[ $fw_dir ]] || fw_dir="$dracutsysrootdir/lib/firmware/updates:$dracutsysrootdir/lib/firmware:$dracutsysrootdir/lib/firmware/$kernel"
 [[ $tmpdir_l ]] && tmpdir="$tmpdir_l"
+[[ $tmpdir ]] || tmpdir="$TMPDIR"
```

Comment:
I am still very uncomfortable about this. Users should really explicitly use `--tmpdir "$TMPDIR"` here. Is that so important for you?

Reply:
Not really anymore, since patched dracut works without changing tmpdir. I can drop this line.

In general though it's good idea to avoid public directories /tmp and /var/tmp if more private choices (like /run/user/UID) exist, so I'd still encourage you to make this change, perhaps with some advertising of the fact in release notes.

Reply:
In any case, it makes sense to make a new PR for the TMPDIR part. This is a bugfix but that PR is requesting a new feature.